mod_regression/controllers.py

```diff
@@ -195,14 +198,22 @@ def toggle_active_status(regression_id):
     })
 
 
-@mod_regression.route('/test/<regression_test_output_id>/download', methods=['GET'])
-def test_result_file(regression_test_output_id):
+@mod_regression.route('/test/<regression_test_output_id>/download/<multiple_files>', methods=['GET'])
+def test_result_file(regression_test_output_id, multiple_files):
     """View the output files of the regression test."""
-    rto = RegressionTestOutput.query.filter(RegressionTestOutput.id == regression_test_output_id).first()
-    if rto is None:
-        g.log.error(f'requested regression test output with id: {regression_test_output_id} not found!')
-        abort(404)
-    return serve_file_download(rto.filename_correct, 'TestResults', 'regression-download')
+    if multiple_files == 'True':
```

Comment:
IMO, should we make `multiple_files` an optional field to ensure backward compatibility. Also, I'm not very sure if the exact 'True' should be kept as the parameter.

cc/ @canihavesomecoffee

Reply:
I'm not so sure why you'd want to keep that optional... It's not like we need to care about broken links for these downloads. As long as all calls inside our program are updated to use it, it should be fine...

Reply:
Yeah. If we are simultaneously updating the calls, then good.

Reply:
Well, at least one of the uses is further down in this PR :)

Let's check if there's others.

Reply:
@kvshravan Can you make sure that this change doesn't break https://github.com/CCExtractor/sample-platform/blob/master/tests/test_regression/TestControllers.py#L52 (or that this test is adapted)?